sdk/storage/azure-storage-queue/src/samples/java/com/azure/storage/queue/QueueAsyncJavaDocCodeSamples.java

```diff
@@ -145,7 +145,7 @@ public void enqueueMessageAsync() {
      */
     public void enqueueMessageBinaryDataAsync() {
         // BEGIN: com.azure.storage.queue.queueAsyncClient.sendMessage#BinaryData
-        client.sendMessage(BinaryData.fromString("Hello, Azure")).subscribe(
+        client.sendMessage(BinaryData.fromString("Hello, Azure").toString()).subscribe(
```

Comment:
The issue may be slightly incorrect as there are two `sendMessage` APIs, one accepts `String` and the other accepts `BinaryData`.

https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueClient.java#L419

https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueClient.java#L442

Reply:
@alzimmermsft I have updated